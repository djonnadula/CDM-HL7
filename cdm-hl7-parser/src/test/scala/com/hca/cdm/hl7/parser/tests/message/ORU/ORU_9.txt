MSH|^~\&||COCAR|||201701130201||ORU^R01|MT6_COCAR_ORU_ARGTPACS.1.1180582|P|2.4
PID|1||I000356699|I279000|KIPKER KEPLER^JAKET^^^^||19451105|F||||||||||I00970395034|060-00-5601
PV1|1|I|DIS^I.400^A||||TANZA^Tanimu^Zakari^^^MD^^^^^^^214-217-1900|.SELF^REFERRED^SELF^^|NOPTH^Nophsker^Theodore^R^DO||||||||TANZA^Tanimu^Zakari^^MD^^^^^^^214-217-1900|IN|||||||||||||||||||||COCAR||DIS|||||||||||DILER^Dilley^Ermilo^Duane^MD~GRAMI^Graceffo^Michael^A^MD~OSOAY^Osowo^Ayodele^T^MD~YANWE01^Yan^Wei^^^MD
ORC|RE||||S|N|||||||COCAR^ARL
OBR|1|001922764&N&N| - CHEST 1 VIEW PORTABLE|CHEP^ - CHEST 1 VIEW PORTABLE^RAD|R|201701030000|201701030747|201701030750||||||||DILER|817-465-5881|.SELF~DILER~YANWE01|||||||F||||||copd|L.PHY.MAS&&&&&&^201701030750||||201701030400
ZUR|1|I772951
ZUR|2|I001922764
ZUR|3|FWDCGEWEB02.HCA.CORPAD.NET
ZUR|4|COCAR
OBX|1|TX|001922764^ - CHEST 1 VIEW PORTABLE^CHEP|1| ~ MEDICAL CENTER OF ARLINGTON Name: KIPLER KELLER,JANET ~ 3301 MATLOCK ROAD Phys: Dilley,Ermilo Duane MD ~ ARLINGTON,TX 76015 DOB: 11/05/1945 Age: 71 Sex: F~ Acct: I00970395034 Loc: I.400 A ~ PHONE #: 817-472-4873 Exam Date: 01/03/2017 Status: DIS IN ~ FAX #: 469-713-8222 Radiology No: ~ Unit No: I000356699 ~ ~ ~ ~ EXAMS: CPT CODE: 001922764 CHEST 1 VIEW PORTABLE 71010 ~ ~ COMPARISON: 12/31/16 0446 hours ~ ~ CLINICAL INFORMATION: COPD. ~ ~ FINDINGS: ~ ~ Bones are similar and appropriate for age including evidence for a ~ remote vertebroplasty in the upper mid thoracic vertebral body. ~ Mildly enlarged cardiomediastinal silhouette is stable. ~ ~ Lungs display stable emphysematous changes. There is a partially ~ calcified nodule in the right midlung field measuring approximately ~ 10 x 12 mm, unchanged in the interval and probably represents a ~ calcified granuloma in the right lower lobe. Previously described ~ reticular interstitial lung opacities have improved in the interval ~ and most consistent with resolving edema/fluid overload superimposed ~ on previously described pulmonary emphysema and chronic interstitial ~ changes. Blunted costophrenic angles remain similar. ~ ~ IMPRESSION: ~ ~ 1. FINDINGS CONSISTENT WITH RESOLVING EDEMA/INTERSTITIAL PNEUMONITIS ~ SUPERIMPOSED ON CHRONIC CHANGES, AS DESCRIBED ABOVE. ~ ~ 2. STABLE MILD CARDIOMEGALY. ~ ~ 3. EVIDENCE FOR OLD GRANULOMATOUS DISEASE. ~ ~ AFC- 99053 ~ ~ RL-501 ~ ______________________________________ ~ ~ Electronically signed by: M SALAM ~ Date: 01/03/17 ~ Time: 07:50 ~ ~ ~ ** Electronically Signed by M.D. SALAM MAQBOOL ** ~ ** on 01/03/2017 at 0750 ** ~ Reported and signed by: SALAM MAQBOOL, M.D. ~ ~ ~ ~ ~ ~ ~PAGE 1 Signed Report (CONTINUED) ~ ~ MEDICAL CENTER OF ARLINGTON Name: KIPLER KELLER,JANET ~ 3301 MATLOCK ROAD Phys: Dilley,Ermilo Duane MD ~ ARLINGTON,TX 76015 DOB: 11/05/1945 Age: 71 Sex: F~ Acct: I00970395034 Loc: I.400 A ~ PHONE #: 817-472-4873 Exam Date: 01/03/2017 Status: DIS IN ~ FAX #: 469-713-8222 Radiology No: ~ Unit No: I000356699 ~ ~ ~ ~ EXAMS: CPT CODE: 001922764 CHEST 1 VIEW PORTABLE 71010 ~ <CONTINUED> ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~CC: SELF REFERRED; Ermilo Duane Dilley MD; Wei Yan MD ~ ~Dictated Date/Time: 01/03/2017 (0747)~Technologist: MAIN XRAY PORTABLE B; PHONG HO RT(R) ~Transcribed Date/Time: 01/03/2017 (0750)~Transcriptionist: L.PHY.MAS ~Electronic Signature Date/Time: 01/03/2017 (0750)~Printed Date/Time: 01/13/2017 (0201) BATCH NO: N/A ~ ~PAGE 2 Signed Report