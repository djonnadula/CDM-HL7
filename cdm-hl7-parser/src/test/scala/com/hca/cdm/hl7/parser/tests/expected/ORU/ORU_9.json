{
  "0000.COMN": {
    "sending_facility": "COCAR",
    "message_control_id": "MT6_COCAR_ORU_ARGTPACS.1.9879817",
    "medical_record_num": "I5012456699",
    "medical_record_urn": "I875617",
    "patient_account_num": "I12541241232"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": null,
    "sending_facility": "COCAR",
    "receiving_application": null,
    "receiving_facility": null,
    "date_time_of_message": "201402210201",
    "security": null,
    "message_type": {
      "message_code": "ORU",
      "trigger_event": "R01"
    },
    "message_control_id": "MT6_COCAR_ORU_ARGTPACS.1.9879817",
    "processing_id": "P",
    "version_id": "2.4"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": "I5012456699",
    "alternate_patient_id": "I875617",
    "patient_name": {
      "family_name": "LIPSITCKET ORION",
      "given_name": "RACKETBACK",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19540931",
    "administrative_sex": "F",
    "patient_alias": null,
    "race": null,
    "patient_address": null,
    "county_code": null,
    "phone_num_home": null,
    "phone_num_business": null,
    "primary_language": null,
    "marital_status": null,
    "religion": null,
    "patient_account_num": "I12541241232",
    "ssn_num_patient": "999-99-9999"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "DIS",
      "room": "I.400",
      "bed": "A"
    },
    "admission_type": null,
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "STANT",
      "family_name": "Laribou",
      "given_name": "Caribou",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": "MD",
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "999-999-9999"
    },
    "referring_doctor": {
      "person_id": ".SELF",
      "family_name": "REFERRED",
      "given_name": "SELF",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null
    },
    "consulting_doctor": {
      "person_id": "FLOPB",
      "family_name": "Flopper",
      "given_name": "Betty",
      "second_and_further_given_names_or_initials_thereof": "R",
      "suffix": "DO"
    },
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": null,
    "ambulatory_status": null,
    "vip_indicator": null,
    "admitting_doctor": {
      "person_id": "STANT",
      "family_name": "Laribou",
      "given_name": "Caribou",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": "MD",
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": "999-999-9999"
    },
    "patient_type": "IN",
    "visit_num": null,
    "financial_class": null,
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": "COCAR",
    "bed_status": null,
    "account_status": "DIS",
    "pending_location": null,
    "prior_temporary_location": null,
    "admit_date_time": null,
    "discharge_date_time": null,
    "current_patient_balance": null,
    "total_charges": null,
    "total_adjustments": null,
    "total_payments": null,
    "alternate_visit_id": null,
    "visit_indicator": null,
    "primary_care_doctor": [
      {
        "person_id": "QESDF",
        "last_name": "Filly",
        "first_name": "Merlio",
        "middle_name": "Gary",
        "suffix": "MD"
      },
      {
        "person_id": "ARRFG",
        "last_name": "Racecar",
        "first_name": "Portnow",
        "middle_name": "A",
        "suffix": "MD"
      }
    ]
  },
  "0004.ORC": {
    "order_control": "RE",
    "placer_order_num": null,
    "filler_order_num": null,
    "placer_group_num": null,
    "order_status": "S",
    "response_flag": "N",
    "quantity_timing": null,
    "parent_order": null,
    "date_time_of_transaction": null,
    "entered_by": null,
    "verified_by": null,
    "ordering_provider": null,
    "enterer_location": {
      "point_of_care": "COCAR",
      "room": "ARL"
    }
  },
  "0005.OBR": {
    "set_id": "1",
    "placer_order_num": "001922764&N&N",
    "filler_order_num": " - CHEST 1 VIEW PORTABLE",
    "universal_service_id": {
      "id": "CHEP",
      "text": " - CHEST 1 VIEW PORTABLE",
      "name_of_coding_system": "RAD"
    },
    "priority": "R",
    "requested_date_time": "201701030000",
    "obsv_date_time_num": "201701030747",
    "obsv_end_date_time_num": "201701030750",
    "collection_volume": null,
    "collector_id": null,
    "specimen_action_code": null,
    "danger_code": null,
    "relevant_clinical_information": null,
    "specimen_received_date_time": null,
    "specimen_source": null,
    "ordering_provider": "QESDF",
    "order_callback_phone_num": "817-465-5881",
    "placer_field_1": ".SELF~QESDF~YANWE01",
    "placer_field_2": null,
    "filler_field_1": null,
    "lab_site": null,
    "results_rpt_status_chng_date_time": null,
    "charge_to_practice": null,
    "diagnostic_service_section_id": null,
    "result_status": "F",
    "parent_result": null,
    "quantity_timing": null,
    "result_copies_to": null,
    "parent_results_obsv_id": null,
    "transportation_mode": null,
    "reason_for_study": "copd",
    "principal_result_interpreter": {
      "name": {
        "id_num": "L.PHY.MAS",
        "family_name": null,
        "given_name": null,
        "second_and_further_given_names_or_initials_thereof": null,
        "suffix": null,
        "prefix": null,
        "degree": null,
        "source_table": "201701030750"
      }
    },
    "assistant_result_interpreter": null,
    "technician": null,
    "transcriptionist": null,
    "scheduled_date_time": "201701030400"
  },
  "0006.ZUR": {
    "set_id": "1",
    "information": "I772951"
  },
  "0007.ZUR": {
    "set_id": "2",
    "information": "I001922764"
  },
  "0008.ZUR": {
    "set_id": "3",
    "information": "FWDCGEWEB02.HCA.CORPAD.NET"
  },
  "0009.ZUR": {
    "set_id": "4",
    "information": "COCAR"
  }
}